user-guide/Troubleshooting/Procedures/Remote_Connecting_Issues.md

```diff
@@ -0,0 +1,20 @@
+---
+uid: Remote_Connecting_Issues
+---
+
+# Remote Connecting Issues
+If you're encountering issues with remote connections, here are some common problems and potential solutions:
+
+## Check CloudGateway Status
+One of the common issues why the remote access is unavailable is due to the CloudGateway DxM not being up. In this way it is impossible to connect with the DataMiner Agent using remote instances. 
+The following possible options may be the solution to the problem:
+1. Verify whether CloudGateway DxM is running. If not, start it.
+2. Restart the CloudGateway DxM to refresh its state.
+3. Consider upgrading the CloudGateway DxM (and other DxMs in the Cloud Pack) to the latest version. Mostly an update can resolve connectivity issues.
```

Comment:
I think md will number them correctly, in that case let's just put `1.` everywhere.
```suggestion
1. Make sure to have the most recent versions of the DxMs installed to avoid dealing with issues that might have been fixed yet.
```
Maybe add a link to the change log here, and how to update dxms (docs)